app/lib/email.ts

```
@@ -45,7 +46,11 @@ export async function sendEmail(
   try {
     await client.send(command)
   } catch (e) {
-    if (
+    if (e instanceof MessageRejected && hostname == 'localhost') {
```

This must still throw in production.

---

That should be okay though since the the hostname should be different right?

---

Hmm true, think I should just let it throw normally and remove this condition?

---

That's fine with me.